kihuha/iReporter
# #163354245 - Update signup UI and JS

#### What does this PR do?
Contains an updated sign up page UI and script

#### Description of the task to be completed
- Provide the users with more information on data values that are mandatory (required)
- Include required attribute to the phone number input

#### How should this be manually tested
Navigate to `https://kihuha.github.io/iReporter/ui/signup.html` and fill in the form with correct data. If the entered values are correct, the page will redirect to the login page from where you can login. If the values are invalid, an alert banner appears showing the values that are invalid.

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/163354245
